Sources/HTTP/HTTPMethod.swift

```diff
@@ -0,0 +1,81 @@
+// This source file is part of the Swift.org Server APIs open source project
+//
+// Copyright (c) 2017 Swift Server API project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+//
+
+/// HTTP method structure
+public struct HTTPMethod: RawRepresentable {
+    /// HTTP method
+    public let rawValue: String
+    
+    /// Creates an HTTP method
+    public init(rawValue: String) {
+        self.rawValue = rawValue.uppercased()
+    }
+    
+    /// Creates an HTTP method
+    public init(_ rawValue: String) {
```

Comment:
Those are two duplicate inits, I think there should be only one of them. The first if we keep that as a `RawRepresentable` and the second, if not.